src/applications/edu-benefits/10203/manifest.json

```diff
@@ -3,7 +3,7 @@
   "entryFile": "./edu-benefits-entry.jsx",
   "entryName": "10203-edu-benefits",
   "rootUrl": "/education/apply-for-education-benefits/application/10203",
-  "production": true,
+  "production": false,
```

Comment:
Sorry this is kinda misleading but the prod feature flag is actually `template.vagovprod: false`. I'm not sure this `production` values has an effect anymore. See this guide for reference, https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/new-application